### ISSUE TITLE
windows native email, nvda does not announce messages as soon as you open the app 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12833]
### Steps to reproduce:
open the email app, native windows app 
### Actual behavior:
nvda advertises loading, or advertises just list. 
### Expected behavior:
I expect nvda to advertise loading if messages are still loading from the server, and as soon as the messages finish loading nvda should announce the first hover message.
What seems to happen is that nvda manages to capture the first information that is loading, this means that the messages are not in the box yet, the email app is loading.
However when messages are in the folder, and a message effectively already has focus, nvda cannot capture this, we can press arrows at will and nvda is muted.
The only way to get the messages read is by pressing tabe and then shift + tabe to put nvda's focus back on the mailing list, then everything works fine.
Perhaps to reproduce this problem you have to reproduce using a slower connection so that when you open the e-mails app the messages are not yet ready for browsing. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
Nnvda.exe, NVDA alpha-23729,f8f62b04

#### Windows version:
11 10.0.22000.184
#### Name and version of other software in use when reproducing the issue:
HxOutlook.exe, microsoft.windowscommunicationsapps 16005.14326.20206.0

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[email1.txt](https://github.com/nvaccess/nvda/files/7148109/email1.txt)

**Comment by ahmad-alajaj** [comment]
Hi,
The problem you are talking about does not always happen
Although I don't use it, but I tried it, the problem actually happens sometimes 

Are you using Windows 10 App Essentials add-on?

**Comment by fernando-jose-silva** [comment]
For me this problem is so frequent that it is possible to say that it occurs 99% of times I open the email app.
When I use nvda without add-ons the problem also occurs.

**Comment by feerrenrut** [comment]
I can't reproduce this issue, but I do not have many emails in the Mail app. The log provided doesn't seem to demonstrate the issue (it seems like mail was already open when starting NVDA?).

Please reproduce the issue with the UIA (advanced) logging option enabled, and debug logging. Please do this with all add-ons disabled. Collect the log and attach it to this issue.

**Comment by fernando-jose-silva** [comment]
unfortunately I can still reproduce the problem every day.
The email app was closed, as now in the log I send with your directions.
thanks. 
[email.txt](https://github.com/nvaccess/nvda/files/7661075/email.txt)

**Comment by seanbudd** [comment]
We have removed this milestone as there is a workaround, and it might not be able to be included in 2022.1. It's possible this issue might need to be reported upstream to microsoft.

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this problem with NvDA 2024.4.1?

**Comment by fernando-jose-silva** [comment]
unfortunately I am no longer a user of this email client, thank you very much.Enviado do meu iPhoneEm 11 de nov. de 2024, à(s) 16:42, Adriani90 ***@***.***> escreveu:﻿
@fernando-jose-silva are you still having this problem with NvDA 2024.4.1?

**Comment by Adriani90** [comment]
Ok, it seems native Windows mail app reaches end of life on December 31 2024 after which people will not be able anymore to send and receive messages in this app. You can install the new Outlook for Windows if you still want the native email experience. Closing as abandoned.
For more details, see
https://support.microsoft.com/en-us/office/outlook-for-windows-the-future-of-mail-calendar-and-people-on-windows-11-715fc27c-e0f4-4652-9174-47faa751b199
